pom.xml

```diff
@@ -0,0 +1,70 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>jaxrs.example</groupId>
+  <artifactId>jaxrs.example</artifactId>
+  <version>0.0.2-SNAPSHOT</version>
+  <packaging>war</packaging>
+  
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
+  </properties>
+
+  <!-- DistributionManagement -->
+  <distributionManagement>
+    <repository>
+      <id>jboss-releases-repository</id>
+      <name>JBoss Releases Repository</name>
+      <url>https://repository.jboss.org/nexus/service/local/staging/deploy/maven2/</url>
+    </repository>
+    <snapshotRepository>
+      <id>jboss-snapshots-repository</id>
+      <name>JBoss Snapshots Repository</name>
+      <url>https://repository.jboss.org/nexus/content/repositories/snapshots/</url>
+    </snapshotRepository>
+  </distributionManagement>
```

Comment:
Is there a reason we need these?

Reply:
Well, I don't know. At a RESTEasy meeting when you were away, I asked about deploying directly to JBoss nexus. I had copied stuff in the course of a release, but I don't remember doing it directly. Anyway, Jim mentioned this addition, and it worked, so it's there. I'm guessing setting the parent to org.jboss:jboss-parent would work?

Reply:
Ok, I set the parent to org.jboss:jboss-parent.

Reply:
No, I need something in the pom.xml of  the generated project. I put back <distributionManagement>.